WordCounter/Histogram.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WordCounterProject
+{
+    class Histogram
+    {
+        public Histogram() { }
```

Comment:
If this is only going to hold static methods you shouldn't put a constructor here.